test/ctia/http/generative/specs.clj

```diff
@@ -0,0 +1,87 @@
+(ns ctia.http.generative.specs
```

Comment:
This will probably grow and will need to be split up later, but it is concise at the moment, so its in one file.